package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "redux-bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
```

Comment:
![image](https://user-images.githubusercontent.com/31897434/76875582-086a5180-6871-11ea-8da1-6e2bf375cce0.png)
![image](https://user-images.githubusercontent.com/31897434/77427491-cee89780-6dd6-11ea-952f-0e132dedf985.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/
You missed those 👆 
Kindly add it and resubmit for re-review.

Reply:
We are not working alone, I was having some issues with my create-react-app. Thanks